### ISSUE TITLE
How to extract the semantic information for the replica data

### ISSUE THREAD

**Issue by Runsong123** [QianyiWu/objectsdf_plus#3]
Dear author,

Thanks for your previous help. I'm still reading and learning about your code framework.
I notice that the replica data used in your paper is consistent with MonoSDF and you have kindly provided the instance ID. Now, I only want to obtain the semantic label for current data. I am wondering if there is any easy way to get the semantic labels. 

Any help would be appreciated.

Bests,
Runsong

**Comment by DaLi-Jack** [comment]
Yes, I'd like to know too, any help would be greatly appreciated!

**Comment by QianyiWu** [comment]
Hi, @DaLi-Jack @Runsong123 

These code snippets might help you [here](https://github.com/kxhit/vMAP/tree/master/data_generation). Please refer to Semantic-NeRF for more details.

**Comment by DaLi-Jack** [comment]
okk! I will try this later, thanks for your help!

**Comment by Runsong123** [comment]
Got it. Thanks!
